LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Does FreeCAD have any policy on licenses?
I saw that the license is the same as [here](https://gitlab.com/daviddaish/freecad_docker_env/-/blob/master/LICENSE) so I'm also asking to make sure that the choice of license was intentional and that it didn't just _slip along_

Reply:
GPL is avoided to make it possible to use with propritary tools and libraries. Most of freecad is LGPL.
MIT, BSD and similar is ok too.